.env.example

```diff
@@ -26,6 +26,11 @@ GREENHOUSE_API_TOKEN='greenhouse-api-token'
 
 REDIS_URL=redis://localhost:6379/0
 
+ADOBE_LAUNCH_URL=''
+PRIMARY_CATEGORY=''    #example value='biz:api:adp:developer'
```

Comment:
Let's rename this also to ADOBE_LAUNCH_PRIMARY_CATEGORY

Reply:
Renamed it.